app/vmctl/opentsdb/exampleOutput.json

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
would be nice to put fixtures in `testdata` folder. See https://dave.cheney.net/2016/05/10/test-fixtures-in-go

Reply:
I honestly put these in as examples of the queries/results themselves. Not really as test fixtures. But I guess they could become those. Good idea!